packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormDropdown/TransferFormDropdown.tsx

```diff
@@ -52,6 +52,18 @@ const TransferFormDropdown: React.FC<TProps> = ({ fieldName, mobileAccountsListR
     const toAccountLoginId =
         location.pathname === '/wallets/cashier/transfer' ? location.state?.toAccountLoginId : undefined;
 
+    const toDefaultAccount = useMemo(
+        () => toAccountList.walletAccounts.find(wallet => wallet.currency === 'USD'),
+        [toAccountList.walletAccounts]
+    );
+    // This sets a 'To transfer' to USD Wallet account to be selected by default when user transfers from a crypto wallet
+    if (!toAccount && toDefaultAccount) {
+        setValues(prev => ({
```

Comment:
This statement is executed "once" only (twice because of React strict mode) for crypto accounts, checked for non-crypto accounts, the statement did not execute as expectd